net.sourceforge.m64py.M64Py.yml

```diff
@@ -0,0 +1,140 @@
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtTest*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtWeb*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtXml*
+  - rm -rf /app/lib/python/site-packages/sdl2/examples
+  - rm -rf /app/lib/python/site-packages/sdl2/test
+
+modules:
+
+  - name: mupen64plus-core
+    buildsystem: simple
+    build-commands:
+      - cd projects/unix && sed -i '321,324d;315d' Makefile && make install PREFIX=/app SHAREDIR=/app/share/mupen64plus VULKAN=0 OSD=0 NEW_DYNAREC=1
+    sources:
+      - type: archive
+        url: https://github.com/mupen64plus/mupen64plus-core/releases/download/2.6.0/mupen64plus-core-src-2.6.0.tar.gz
+        sha256: 7f79f7f56b94dfa09c8f0dc95720cdffc07c722457e18f46ec8adbbb3505d21f
+
+  - name: mupen64plus-rsp-hle
+    buildsystem: simple
+    build-commands:
+      - cd projects/unix && make install PREFIX=/app APIDIR=/app/include/mupen64plus
+    sources:
+      - type: archive
+        url: https://github.com/mupen64plus/mupen64plus-rsp-hle/releases/download/2.6.0/mupen64plus-rsp-hle-src-2.6.0.tar.gz
+        sha256: 8501b97315db80c7f200ee79c5ce34203c5d1e0c61f93f538263fc83f8464a7f
+
+  - name: mupen64plus-input-sdl
+    buildsystem: simple
+    build-commands:
+      - cd projects/unix && make install PREFIX=/app APIDIR=/app/include/mupen64plus
+    sources:
+      - type: archive
+        url: https://github.com/mupen64plus/mupen64plus-input-sdl/releases/download/2.6.0/mupen64plus-input-sdl-src-2.6.0.tar.gz
+        sha256: 3b8629f72bb137fd2bdb9a6cc29ad270c3877f3d40e1f9dae89db24dde0425e4
+
+  - name: mupen64plus-audio-sdl
+    buildsystem: simple
+    build-commands:
+      - cd projects/unix && make install PREFIX=/app APIDIR=/app/include/mupen64plus
+    sources:
+      - type: archive
+        url: https://github.com/mupen64plus/mupen64plus-audio-sdl/releases/download/2.6.0/mupen64plus-audio-sdl-src-2.6.0.tar.gz
+        sha256: 8840434b641fd600f9f79911bba279f284a53eb8e746d83372a7692ba8dac95c
+
+  - name: mupen64plus-video-rice
+    buildsystem: simple
+    build-commands:
+      - cd projects/unix && make install PREFIX=/app APIDIR=/app/include/mupen64plus NO_ASM=1
+    sources:
+      - type: archive
+        url: https://github.com/mupen64plus/mupen64plus-video-rice/releases/download/2.6.0/mupen64plus-video-rice-src-2.6.0.tar.gz
+        sha256: 19f947d858f66aec41a99c0430bd41d6a2dff569be324d13156bd78fb89eadc4  
+
+  - name: mupen64plus-video-GLideN64
+    buildsystem: cmake-ninja
+    subdir: src
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DMUPENPLUSAPI=On
+      - -DUSE_SYSTEM_LIBS=On
+    sources:
+      - type: archive
+        url: https://github.com/gonetz/GLideN64/archive/f4b0755e2722d59fe1c20d7e470f7b8929dcae85.tar.gz
+        sha256: 738ed313ce93f3faca15420cf45549fb51e683e6ab1c2dcd91e42ead1eed8a89
+
+  - name: unrar
+    buildsystem: simple
+    build-commands:
+      - make
+      - make install DESTDIR=/app
```

Comment:
no-autogen: true, DESTDIR in make-args, drop buildsystem: simple

Reply:
Changed.